src/main/java/com/st/utopia/agent/util/RestTemplateResponseErrorHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.st.utopia.agent.util;
+
+import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.web.client.ResponseErrorHandler;
+
+public class RestTemplateResponseErrorHandler implements ResponseErrorHandler {
+	@Override
+	public boolean hasError(ClientHttpResponse httpResponse) {
+		/* By default it would return true and throw a error if it was anything other
```

Comment:
I'd suggest making this a comment on the method, not merely the return value.

Reply:
Will do.